CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to
 - Remove the `-dd` CLI option
   - [#3203](https://github.com/bpftrace/bpftrace/pull/3203)
 #### Fixed
+- Fix segfault of CodegenLLVM::generateProbe()
```

Comment:
Sorry, one more thing. Since this is user-facing, could we go with something like
```suggestion
- Fix segfault for multi-tracepoint probes
```

Reply:
Fixed, thanks!